community/index/src/main/java/org/neo4j/index/internal/gbptree/CleanupJob.java

```diff
@@ -42,6 +42,11 @@
      */
     Exception getCause();
 
+    /**
+     * Mark this job and cleanup all it's resources.
```

Comment:
`Mark this job....` as closed perhaps? No biggie, I do believe that anyone reading that kind of gets what it wants to convey though